src/test/java/uk/gov/hmcts/sptribs/document/content/PreviewDraftOrderTemplateContentTest.java

```diff
@@ -22,20 +31,28 @@ public class PreviewDraftOrderTemplateContentTest {
     @InjectMocks
     private PreviewDraftOrderTemplateContent previewDraftOrderTemplateContent;
 
-    private static final LocalDate ISSUE_DATE = LocalDate.of(2022, 2, 2);
-
     @Test
     public void shouldSuccessfullyPreviewDraftOrderContent() {
```

Comment:
Two more tests can be added
1. completed hearing date being null
2. PreviewDraftOrderContent with no members